Buildings/Controls/OBC/CDL/Continuous/AMIGOWithFOTD.mo

```diff
@@ -0,0 +1,129 @@
+within Buildings.Controls.OBC.CDL.Continuous;
+block AMIGOWithFOTD "AMIGO tuning method for a first order model with time delay"
+  parameter Buildings.Controls.OBC.CDL.Types.SimpleController controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller";
+  parameter Real yHigher = 1 "Higher value for y";
+  parameter Real yLower = 0 "Lower value for y";
+  parameter Real deadBand = 0.5 "Deadband for holding the output value";
+
+  Real y0 "Initial value of the process output";
+  Real u0 "Initial value of the process input";
+  Interfaces.RealInput tau "Normalized time delay"
+    annotation (Placement(transformation(extent={{-140,80},{-100,120}}), iconTransformation(extent={{-140,80},{-100,120}})));
+  Interfaces.RealInput dtOFF "Half-period length of a relay tuner"
+    annotation (Placement(transformation(extent={{-140,-120},{-100,-80}}),iconTransformation(extent={{-140,-120},{-100,-80}})));
+
+  Interfaces.RealInput dtON "Half-period length of a relay tuner"
+    annotation (Placement(transformation(extent={{-140,-72},{-100,-32}}),
+                                                                        iconTransformation(extent={{-140,-78},{-100,-38}})));
```

Comment:
Use camelCase (dtOff and dtOn), and correct formatting.

Reply:
The camelCase is enforced and the formatting is corrected.